docs/source/sysadmin_guide/installation.rst

```diff
@@ -0,0 +1,85 @@
+*********************
+Installation with APT
+*********************
+
+.. note::
+
+  These instructions have been tested on Ubuntu 16.04 only.
+
+Stable Builds
+-------------
+
+Add the Sawtooth Lake repository to your ``/etc/apt/sources.list``:
+
+.. code-block:: console
+
+  $ sudo echo "deb http://repo.sawtooth.me/ubuntu/0.8 xenial universe" >> /etc/apt/sources.list
+
+Add the Sawtooth Lake signing key:
+
+.. code-block:: console
+
+  $ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 6B58B1AC10FB5F63
+
+Update your repositories and install Sawtooth Lake:
+
+.. code-block:: console
+
+  $ sudo apt-get update
+
+.. code-block:: console
+
+  $ apt-get install -y -q \
+    python3-sawtooth-cli \
+    python3-sawtooth-config \
+    python3-sawtooth-intkey \
+    python3-sawtooth-manage \
+    python3-sawtooth-poet-common \
+    python3-sawtooth-poet-core \
+    python3-sawtooth-poet-families \
+    python3-sawtooth-poet-simulator \
```

Comment:
+ python3-sawtooth-poet-cli

Assuming you got this from the `sawtooth-all` Dockerfile, that is probably out of date now too.